pkgs/development/python-modules/pyspotify/default.nix

```diff
@@ -41,7 +41,5 @@ buildPythonPackage rec {
     description = "A Python interface to Spotify’s online music streaming service";
     license = licenses.unfree;
     maintainers = with maintainers; [ lovek323 ];
-    platforms = platforms.unix;
```

Comment:
why did that line go away completely?

Reply:
Because that is the default for pythonPackages and we don't repeat the default.